README.md

```diff
@@ -1,4 +1,6 @@
-This repository contains the Ultimaker package build script for the kernel
+[![pipeline status](https://gitlab.com/olliver/um-kernel/badges/master/pipeline.svg)](https://gitlab.com/olliver/um-kernel/commits/master)
```

Comment:
that's actually olliver is username OR group, and this should be 'ultimaker'. this is a blocker to merge this issue for now imo and wait until next week if we can get the ultimaker moniker. Otherwise we have ultimaker_bv at the moment as fallback.